_vector-search/vector-search-techniques/index.md

```diff
@@ -0,0 +1,72 @@
+---
+
+# Vector search techniques
+
+OpenSearch implements vector search as *k-nearest neighbors*, or *k-NN*, search. k-NN search finds the k neighbors closest to a query point across an index of vectors. To determine the neighbors, you can specify the space (the distance function) you want to use to measure the distance between points.
+
+OpenSearch supports three different methods for obtaining the k-nearest neighbors from an index of vectors:
+
+- [Approximate search]({{site.url}}{{site.baseurl}}/search-plugins/knn/approximate-knn/)) (approximate k-NN, or ANN): Returns approximate nearest neighbors to the query vector. Usually, approximate search algorithms sacrifice indexing speed and search accuracy in exchange for performance benefits such as lower latency, smaller memory footprints, and more scalable search. For most use cases, approximate search is the best option.
+
+- Exact search: A brute-force, exact k-NN search of vector fields. OpenSearch supports the following types of exact search: 
+  - [Exact search with scoring script]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-score-script/): Using a scoring script, you can apply a filter to an index before executing the nearest neighbor search. 
+  - [Painless extensions]({{site.url}}{{site.baseurl}}/search-plugins/knn/painless-functions/): Adds the distance functions as Painless extensions that you can use in more complex combinations. You can use this method to perform a brute-force, exact vector search of an index, which also supports pre-filtering. 
+
+
+Overall, for larger data sets, you should generally choose the approximate nearest neighbor method because it scales significantly better. For smaller data sets, where you may want to apply a filter, you should choose the custom scoring approach. If you have a more complex use case where you need to use a distance function as part of their scoring method, you should use the Painless scripting approach.
+
+## Approximate search
+
+OpenSearch supports the following algorithms (_methods_) for approximate vector search:
+
+- Hierarchical Navigable Small World (HNSW)
+- Inverted File System (IVF)
+
+Additionally, you can choose one of the following libraries (_engines_) that implement these algorithms:
+
+- [Facebook AI Similarity Search (Faiss)](https://github.com/facebookresearch/faiss)
+- Lucene
+- [Non-Metric Space Library (NMSLIB)](https://github.com/nmslib/nmslib) (deprecated)
+
+The following table lists the combinations of search methods and libraries supported by the k-NN engine for approximate vector search.
+
+Method | Engine
+:--- | :---
+HNSW | Faiss, Lucene, NMSLIB (deprecated)
+IVF | Faiss 
+
+For more information, see [Methods and engines]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/knn-methods-engines/) and [Choosing the right method]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/knn-methods-engines/#choosing-the-right-method).
+
+## Engine recommendations
```

Comment:
Lets remove this section - for more details, users can reference the methods and engines section.

Reply:
@jmazanec15 Should we move this into the "Methods and engines" page? This table has information about things not mentioned in the "Methods and engines", like filter options and query latency and quality.

Reply:
Right, I think it makes sense to be moved to that page